### PR TITLE
Bump hako 2.5.0

### PR DESCRIPTION
**Pull request by mgi166**
Bump latest `hako` and create image.
https://rubygems.org/gems/hako/versions/2.5.0

```
% docker build . -t hako-2.5.0 -f hako/2.5.0/Dockerfile
Sending build context to Docker daemon  102.9kB
Step 1/6 : FROM alpine:3.8 AS builder
 ---> 196d12cf6ab1
Step 2/6 : RUN apk add --update --no-cache --virtual .build-deps build-base git       && git clone https://github.com/google/jsonnet.git       && cd jsonnet       && make libjsonnet.so       && mkdir -p /opt/jsonnet       && cp libjsonnet.so /opt/jsonnet/libjsonnet.so       && cp include/libjsonnet.h /opt/jsonnet/libjsonnet.h       && cd ..       && rm -rf jsonnet       && apk del --purge .build-deps
 ---> Using cache
 ---> 88d8df621bd8
Step 3/6 : FROM ruby:2.5-alpine3.8
 ---> 2b1694a122af
Step 4/6 : COPY --from=builder /opt/jsonnet/libjsonnet.so /usr/local/lib/libjsonnet.so
 ---> Using cache
 ---> e534d94d4c50
Step 5/6 : COPY --from=builder /opt/jsonnet/libjsonnet.h /usr/local/include/libjsonnet.h
 ---> Using cache
 ---> a287470dbc41
Step 6/6 : RUN apk --update --no-cache add openssh git zip make docker libstdc++       && apk add --update --no-cache --virtual .build-deps g++       && JSONNET_USE_SYSTEM_LIBRARIES=1 gem install hako -v 2.5.0 -N --no-rdoc --no-ri       && apk del --purge .build-deps
 ---> Using cache
 ---> 90f9f4dfc21a
Successfully built 90f9f4dfc21a
Successfully tagged hako-2.5.0:latest

% docker run --rm -it hako-2.5.0 sh
/ # gem list | grep hako
hako (2.5.0)
```

### CODE REVIEW

File: hako/2.5.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      && cd .. \
+      && rm -rf jsonnet \
+      && apk del --purge .build-deps
+
+FROM ruby:2.5-alpine3.8
+COPY --from=builder /opt/jsonnet/libjsonnet.so /usr/local/lib/libjsonnet.so
+COPY --from=builder /opt/jsonnet/libjsonnet.h /usr/local/include/libjsonnet.h
+RUN apk --update --no-cache add openssh git zip make docker libstdc++ \
+      && apk add --update --no-cache --virtual .build-deps g++ \
+      && JSONNET_USE_SYSTEM_LIBRARIES=1 gem install hako -v 2.5.0 -N --no-rdoc --no-ri \
```
Comment (saorio) [inline hako/2.5.0/Dockerfile]: なんか環境変数にしてしまってもいい気がした
ENV HAKO_VERSION= 2.5.0

**Comment by saorio** [comment]
LGTM

File: hako/2.5.0/Dockerfile (hunk 1 above)
Comment (mgi166) [reply hako/2.5.0/Dockerfile]: Thank you for feedback!
Fix it.

Comment (civitaspo) [reply hako/2.5.0/Dockerfile]: @saorio I understand your point, but each `ENV` line creates a new layer, so I disagree that defining `HAKO_VERSION` as `ENV` for reducing image size.

**Pull request merged**